testautoRIFT.py

```diff
@@ -193,14 +188,16 @@ def runAutorift(I1, I2, xGrid, yGrid, Dx0, Dy0, SRx0, SRy0, CSMINx0, CSMINy0, CS
         obj.xGrid = xGrid
         obj.yGrid = yGrid
 
-
-
+    # FIXME: This assumes the zero values in the image are only outside the valid image "frame",
+    #        but is not true for Landsat-7 after the failure of the Scan Line Corrector, May 31, 2003.
+    #        We should not masked based on zero values in the L7 images as this percolates into SearchLimit{X,Y}
+    #        and prevents autoRIFT from looking at large parts of the images.
     # generate the nodata mask where offset searching will be skipped based on 1) imported nodata mask and/or 2) zero values in the image
-    for ii in range(obj.xGrid.shape[0]):
-        for jj in range(obj.xGrid.shape[1]):
-            if (obj.yGrid[ii,jj] != nodata)&(obj.xGrid[ii,jj] != nodata):
-                if (I1[obj.yGrid[ii,jj]-1,obj.xGrid[ii,jj]-1]==0)|(I2[obj.yGrid[ii,jj]-1,obj.xGrid[ii,jj]-1]==0):
-                    noDataMask[ii,jj] = True
+    # for ii in range(obj.xGrid.shape[0]):
+    #     for jj in range(obj.xGrid.shape[1]):
+    #         if (obj.yGrid[ii,jj] != nodata)&(obj.xGrid[ii,jj] != nodata):
+    #             if (I1[obj.yGrid[ii,jj]-1,obj.xGrid[ii,jj]-1]==0)|(I2[obj.yGrid[ii,jj]-1,obj.xGrid[ii,jj]-1]==0):
+    #                 noDataMask[ii,jj] = True
```

Comment:
These lines of code are driving a large part of the missing data we're seeing in L7 processing. Before removing, we're seeing velocities like:
![image](https://user-images.githubusercontent.com/7882693/179067336-07a3b40f-4dc6-4d41-86f3-51d38950e23f.png)

And commenting out these lines, produces:
![image](https://user-images.githubusercontent.com/7882693/179067466-c445c57f-b821-464e-92e4-99241e5e1ac9.png)

showing a significant improvement. 

Fully commenting this out does, however, cause a 20%-ish increase in runtime because we are not not skipping searching over the nodata regions outside of the image. 

Since this is all happening before autoRIFT is run, we might be able to move it after the preprocessing filters and use the ZeroMask calculated in the filters instead of the image nodata. 

Unfortunately, we're still not getting near as good of results as V1 of the data, with large swaths of the images missing still. The final browse image, for example:
![LE07_L1TP_063018_20040810_20200915_02_T1_X_LE07_L1TP_063018_20040911_20200915_02_T1_G0120V02_P035_IL_ASF_OD](https://user-images.githubusercontent.com/7882693/179067992-3dd36e6a-f1f6-4928-aa60-6ed0dd1ab468.png)

So there's still stuff going on. It's all likely in these workflow scripts, but will take quite a bit of effort to track down

Reply:
Alight, I couldn't figure out how to untangle all the logic associated with this step, so I'm just turning it off for L7 only no (or, really, anytime we'd use the `wallis_fill` preprocessing filter)